site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -235,25 +235,39 @@ const Planner: FC = () => {
     }
   }
 
+  const initializePlanner = () => {
+    if (data.length == 0) {
```

Comment:
Would be good to add a comment here that explains there is a future TODO once we have support for multiple planners: to ensure that a default new year is added only when a new planner is created, not when an existing planner is empty.

Reply:
Will do in the code, thank you!